samples/mapdata/countries/fr/fr-all-all/demo.js

```diff
@@ -2,108 +2,108 @@
-    ['fr-f-lc', 65],
-    ['fr-h-cs', 66],
-    ['fr-e-fi', 67],
-    ['fr-h-hc', 68],
-    ['fr-p-mh', 69],
-    ['fr-g-an', 70],
-    ['fr-n-ag', 71],
-    ['fr-a-br', 72],
-    ['fr-p-cv', 73],
-    ['fr-c-cl', 74],
-    ['fr-e-ca', 75],
-    ['fr-b-gi', 76],
-    ['fr-a-hr', 77],
-    ['fr-v-hs', 78],
-    ['fr-k-he', 79],
-    ['fr-b-ld', 80],
-    ['fr-r-la', 81],
-    ['fr-m-mo', 82],
-    ['fr-p-or', 83],
-    ['fr-o-pc', 84],
-    ['fr-k-po', 85],
-    ['fr-r-my', 86],
-    ['fr-q-sm', 87],
-    ['fr-d-yo', 88],
-    ['fr-c-al', 89],
-    ['fr-c-hl', 90],
-    ['fr-u-ha', 91],
-    ['fr-f-lt', 92],
-    ['fr-s-oi', 93],
-    ['fr-v-rh', 94],
-    ['fr-n-ta', 95],
+    ['fr-bre-mb', 0],
+    ['fr-pdl-vd', 1],
+    ['fr-occ-ad', 2],
+    ['fr-pac-vc', 3],
+    ['fr-ges-hm', 4],
+    ['fr-ges-mr', 5],
+    ['fr-hdf-no', 6],
+    ['fr-occ-hp', 7],
+    ['fr-cvl-in', 8],
+    ['fr-naq-vn', 9],
+    ['fr-naq-dd', 10],
+    ['fr-naq-cm', 11],
+    ['fr-pac-am', 12],
+    ['fr-pac-vr', 13],
+    ['fr-pac-ap', 14],
+    ['fr-ara-ai', 15],
+    ['fr-hdf-as', 16],
+    ['fr-pac-bd', 17],
+    ['fr-occ-av', 18],
+    ['fr-occ-ga', 19],
+    ['fr-ges-ab', 20],
+    ['fr-bfc-co', 21],
+    ['fr-bfc-sl', 22],
+    ['fr-cvl-ch', 23],
+    ['fr-naq-cr', 24],
+    ['fr-pdl-ml', 25],
+    ['fr-naq-ds', 26],
+    ['fr-naq-ct', 27],
+    ['fr-ara-dm', 28],
+    ['fr-ara-ah', 29],
+    ['fr-nor-eu', 30],
+    ['fr-idf-es', 31],
+    ['fr-cvl-el', 32],
+    ['fr-occ-hg', 33],
+    ['fr-idf-hd', 34],
+    ['fr-naq-hv', 35],
+    ['fr-pdl-st', 36],
+    ['fr-cvl-il', 37],
+    ['fr-ara-is', 38],
+    ['fr-bfc-ju', 39],
+    ['fr-bfc-hn', 40],
+    ['fr-ara-lr', 41],
+    ['fr-occ-tg', 42],
+    ['fr-occ-lo', 43],
+    ['fr-naq-lg', 44],
+    ['fr-occ-lz', 45],
+    ['fr-bre-iv', 46],
+    ['fr-ges-mm', 47],
+    ['fr-ges-ms', 48],
+    ['fr-bfc-ni', 49],
+    ['fr-naq-cz', 50],
+    ['fr-ara-pd', 51],
+    ['fr-occ-ge', 52],
+    ['fr-naq-pa', 53],
+    ['fr-ara-sv', 54],
+    ['fr-idf-se', 55],
+    ['fr-idf-vp', 56],
+    ['fr-idf-ss', 57],
+    ['fr-idf-vm', 58],
+    ['fr-hdf-so', 59],
+    ['fr-bfc-tb', 60],
+    ['fr-bfc-db', 61],
+    ['fr-idf-vo', 62],
+    ['fr-ges-vg', 63],
+    ['fr-idf-yv', 64],
+    ['fr-cvl-lc', 65],
+    ['fr-cor-cs', 66],
+    ['fr-bre-fi', 67],
+    ['fr-cor-hc', 68],
+    ['fr-nor-mh', 69],
+    ['fr-ges-an', 70],
+    ['fr-occ-ag', 71],
+    ['fr-ges-br', 72],
+    ['fr-nor-cv', 73],
+    ['fr-ara-cl', 74],
+    ['fr-bre-ca', 75],
+    ['fr-naq-gi', 76],
+    ['fr-ges-hr', 77],
+    ['fr-ara-hs', 78],
+    ['fr-occ-he', 79],
+    ['fr-naq-ld', 80],
+    ['fr-pdl-la', 81],
+    ['fr-ges-mo', 82],
+    ['fr-nor-or', 83],
+    ['fr-hdf-pc', 84],
+    ['fr-occ-po', 85],
+    ['fr-pdl-my', 86],
+    ['fr-nor-sm', 87],
+    ['fr-bfc-yo', 88],
+    ['fr-ara-al', 89],
+    ['fr-ara-hl', 90],
+    ['fr-pac-ha', 91],
+    ['fr-cvl-lt', 92],
+    ['fr-hdf-oi', 93],
+    ['fr-ara-rh', 94],
+    ['fr-occ-ta', 95],
     ['undefined', 96],
```

Comment:
Undefined key? Is this a bug with the generator? There are also other cases like this. It is not new to this PR, but should be looked into.

Reply:
The `undefined` values are generated from border and separator lines. I have checked that removing those entries will have no effect on the demos, so I will check if this could be fixed in the demos generator to avoid manual fixing.

Reply:
Done ✔️